src/content/release/breaking-changes/uiscenedelegate.md

```diff
@@ -0,0 +1,246 @@
+---
+title: UISceneDelegate adoption
+description: >
+  A guide for Flutter iOS developers to adopt Apple's UISceneDelegate protocol.
+---
+
+## Summary
+
+Apple is requiring iOS developers to adopt the UISceneDelegate protocol which
+changes the order of initialization when applications launch. Certain usages of
+Flutter inside `application:didFinishLaunchingWithOptions:` on iOS will have to
+be updated.
+
+## Background
+
+Most Flutter users will not have custom logic inside of
+`application:didFinishLaunchingWithOptions:`. Those apps will not need to do any
+code migration. In most cases Flutter will automatically migrate the Info.plist.
+
+Apple is requiring the adoption of the `UISceneDelegate` which reorders the
+initialization of iOS apps. After a `UISceneDelegate` is specified,
+initialization of the Storyboard will be delayed until after
+`application:didFinishLaunchingWithOptions:`. That means
+`UIApplicationDelegate.window` and
+`UIApplicationDelegate.window.rootViewController` cannot be accessed from
+`application:didFinishLaunchingWithOptions:`.
+
+Apple is driving the adoption of the `UISceneDelegate` API since it allows apps
+to have multiple instances of their UIs, like multitasking on iPadOS.
+
+Previously, Flutter’s documentation would suggest that
+`application:didFinishLaunchingWithOptions:` was a good place to set up platform
+channels for users that want easy interop between the host application and
+Flutter. That is no longer a reliable place to register these platform channels
+since the Flutter engine won’t have been created yet.
+
+## Migration guide
+
+### Info.plist migration
+
+`UISceneDelegate`s must be specified in an app’s Info.plist or specified in
+`application:configurationForConnectingSceneSession:options:`. The Flutter tool
+will attempt to automatically edit the Info.plist if no `UISceneDelegate` is
+specified, so nothing may be required beyond running `flutter run` or `flutter
+build` again. Projects can be manually upgraded by adding the following to the
+Info.plist. `FlutterSceneDelegate` is the new class in the Flutter framework
+that users can use as the `UISceneDelegate`.
+
+```plist
```

Comment:
Seems like this is breaking the tests, maybe try `xml`?
```suggestion
```xml
```

Reply:
done